src/scenes/Modals/SettingsModal/SettingsRowDescription.tsx

```diff
@@ -0,0 +1,7 @@
+import styled from 'styled-components';
+
+import { BaseM } from '~/components/core/Text/Text';
+
+export default styled(BaseM)`
+  max-width: 380px;
+`;
```

Comment:
Allows description text to be better aligned

**Before**
<img width="546" alt="image" src="https://user-images.githubusercontent.com/12162433/208528207-edba94ff-4dad-4c6d-a0ff-a1c750cc8ad9.png">


**After**
<img width="552" alt="Screen Shot 2022-12-19 at 4 31 34 PM" src="https://user-images.githubusercontent.com/12162433/208528081-420d744a-9717-4a33-a1ea-325483c5caf7.png">